i18n/en.json

```
@@ -1912,6 +1914,9 @@
   "delete_post.post": "Post",
   "delete_post.question": "Are you sure you want to delete this {term}?",
   "delete_post.warning": "This post has {count, number} {count, plural, one {comment} other {comments}} on it.",
+  "demote_to_user_modal.demote": "Demote",
+  "demote_to_user_modal.desc": "This action demotes the user {username} to a guest. This action will restrict the user's ability to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to demote user {username} to guest?",
```

nit: avoid repeating "This action"

```suggestion
  "demote_to_user_modal.desc": "This action demotes the user {username} to a guest. It will restrict the user's ability to join public channels and interact with users outside of the channels they are currently members of. Are you sure you want to demote user {username} to guest?",
```

---

@wiersgallak? :point_up:

---

Makes sense.  Thanks